python/paddle/tensor/creation.py

```diff
@@ -2577,7 +2577,9 @@ def meshgrid(*args, **kwargs):
         **kwargs (optional): Currently, only accept name in **kwargs
             The default value is None. Normally there is no need for
             user to set this property. For more information, please refer to :ref:`api_guide_Name`.
-
+        indexing (Optional[str]) : the indexing mode, either “xy” or “ij”, defaults to “ij”.If “xy” is selected, the first dimension corresponds to the cardinality
```

Comment:
```suggestion
        indexing (str, optional) : the indexing mode, either “xy” or “ij”, defaults to “ij”.If “xy” is selected, the first dimension corresponds to the cardinality
```

- `Optional[T]` 等价于 `T | None`，是类型上允许传 `None`
- `, optional` 代表参数有默认值，可以不传

Reply:
支持传None，已修改